docs/en/usage/python.md

```diff
@@ -286,33 +295,33 @@ Explorer API can be used to explore datasets with advanced semantic, vector-simi
 
 [Explorer](../datasets/explorer/index.md){ .md-button }
 
-
 ## Using Trainers
 
 `YOLO` model class is a high-level wrapper on the Trainer classes. Each YOLO task has its own trainer that inherits from `BaseTrainer`.
 
 !!! Tip "Detection Trainer Example"
+    ````
```

Comment:
I didn't catch this locally, but the extra indents needs to be fixed manually (mdformat worked as expected because it wasn't smart enough to fix the indent - mdformat-mkdocs might have been though)